packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -434,6 +409,7 @@ export class Outbox {
 		const mainBatch: IBatchCheckpoint = this.mainBatch.checkpoint();
 		return {
 			mainBatch,
+			idAllocationBatch: this.idAllocationBatch.checkpoint(),
```

Comment:
This will result in ID Compressor ops hitting default case in a switch inside of ContainerRuntime.rollback(). That said, same is true with blob attach ops.
Maybe the flow here - when whoever uses this scenario (currently - only Word) hits these asserts, we would need to implement them? And for now adding it here means - we ensure alarms fire when such usage comes?

Not sure - @anthony-murphy, what are your thoughts?

Reply:
Well, if this is tied to rollback, I might argue that IdCompressor should have the semantics of a UUID generator: there's no need to rollback (we've merely bumped a counter). So, maybe I should revert this.

Reply:
@vladsud actually AFAICT, checkpoint is only used in one place (containerRuntime.ts) and only the main batch is extracted from the return value and the rest is ignored. There aren't any docs to indicate how it's intended to be used, but I'm not sure how the ops would end up in a rollback call. What am I missing?